client/my-sites/themes/style.scss

```diff
@@ -100,31 +99,99 @@
 	padding-left: 25px
 }
 
-.themes__sheet-screenshot {
-	position: absolute;
-	width: 500px;
-	top: 90px;
-	right: 20px;
+.themes__sheet-columns {
+	display: flex;
+	flex-direction: row;
+}
+
+.themes__sheet-column-left {
+	display: flex;
+	width: 50%;
+	flex-direction: column;
+
+}
+
+.themes__sheet-column-right {
+	display: flex;
+	width: 50%;
+	flex-direction: column;
+}
+
+
+.themes__sheet-action-bar {
+	height: 50px;
+	width: 100%;
+}
+
+.themes_sheet-action-bar-spacer {
+	height: 50px;
+	width: 100%;
+	background-color: white;
+}
+
+.themes__sheet-action-bar button {
+	margin: 2px;
+}
+
+.themes__sheet-action-bar-cost {
+	font-weight: 600;
+	color: #4AB866;
```

Comment:
Should the color match up with this one? https://github.com/Automattic/wp-calypso/blob/2bfc68dc0d9c0b65a1bf8ad6e28ef314da2d398b/client/components/theme/style.scss#L22